hs_core/signals.py

```diff
@@ -32,3 +32,4 @@
 post_add_reftimeseries_aggregation = django.dispatch.Signal(providing_args=['resource', 'file'])
 post_add_timeseries_aggregation = django.dispatch.Signal(providing_args=['resource', 'file'])
 post_raccess_change = django.dispatch.Signal(providing_args=['resource'])
+post_move_or_rename_file_or_folder = django.dispatch.Signal(providing_args=['resource'])
```

Comment:
I think you may need to add the signals to the rest api.  
https://github.com/hydroshare/hydroshare/blob/develop/hs_core/views/resource_rest_api.py#L617
https://github.com/hydroshare/hydroshare/blob/develop/hs_core/views/resource_folder_hierarchy.py

Reply:
From what I can see, there isn’t a way to edit folder paths from the rest api. I see a post endpoint to create new folders, but maybe I’m missing something.

Reply:
Sorry for being so slow to respond.
https://github.com/hydroshare/hydroshare/blob/develop/hs_rest_api/urls.py#L149